package.json

```diff
@@ -11,5 +11,9 @@
   },
   "devDependencies": {
     "lerna": "^3.22.1"
+  },
+  "dependencies": {
+    "node": "^16.1.0",
+    "postgresql": "0.0.1"
```

Comment:
We shouldn't add any dependencies to the root level package.json because this isn't used on the server

Reply:
This has been removed from the latest commit.